x-pack/plugins/rule_registry/server/utils/create_get_summarized_alerts_fn.ts

```diff
@@ -32,6 +33,7 @@ interface CreateGetSummarizedAlertsFnOpts {
   ruleDataClient: PublicContract<IRuleDataClient>;
   useNamespace: boolean;
   isLifecycleAlert: boolean;
+  formatAlert?: (alert: unknown) => unknown;
```

Comment:
Allows rule types to pass in custom functions for formatting summarized alerts. This allows detection rules to be formatted exactly as they were before.